src/AddItems.js

```diff
@@ -25,11 +25,11 @@ function AddItems(props) {
     const formData = {
       itemName: groceryItem,
       frequency: itemFreq,
-      lastPurchaseDate: null,
+      purchaseDates: [null],
```

Comment:
Better to just do empty array here, I think.
```suggestion
      purchaseDates: [],
```